src/vs/editor/common/config/editorOptions.ts

```diff
@@ -2748,7 +2748,7 @@ export type EditorLightbulbOptions = Readonly<Required<IEditorLightbulbOptions>>
 class EditorLightbulb extends BaseEditorOption<EditorOption.lightbulb, IEditorLightbulbOptions, EditorLightbulbOptions> {
 
 	constructor() {
-		const defaults: EditorLightbulbOptions = { enabled: ShowLightbulbIconMode.OnCode };
+		const defaults: EditorLightbulbOptions = { enabled: ShowLightbulbIconMode.On };
```

Comment:
Let's postpone changing the default until after we do the following (emphasis mine):

> Users that don't have copilot extension installed might see additional lightbulbs on empty lines, if there are extensions that provide code actions for these locations. **We will try this out with various languages and the major language extensions. We will file issues if a language extension provides non-useful code actions as it didn't expect us to ask on empty lines.**